dashboard/test/ui/features/report_abuse.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Report Abuse Form
+  Scenario: Reporting abuse while signed-out
+    Given I am on "http://studio.code.org/report_abuse"
+    And I type "harry@hogwarts.edu" into "#uitest-email"
+    And I select the "13" option in dropdown "uitest-age-selector"
+    And I select the "Other" option in dropdown "uitest-abuse-type"
+    And I type "Mudblood is an offensive term" into "#uitest-abuse-detail"
+    Then I click selector "#uitest-submit-report-abuse" once I see it
+    Then check that the URL matches "https://support.code.org/hc/en-us"
+
+  Scenario: Reporting abuse as a signed-in student
+    Given I create a student named "Harry"
+    Given I am on "http://studio.code.org/report_abuse"
+    And I type "harry@hogwarts.edu" into "#uitest-email"
+    # Age is are already known and passed to the form
+    And I wait until element "#uitest-age-selector" is not visible
+    And I select the "Other" option in dropdown "uitest-abuse-type"
+    And I type "Mudblood is an offensive term" into "#uitest-abuse-detail"
+    Then I click selector "#uitest-submit-report-abuse" once I see it
+    Then check that the URL matches "https://support.code.org/hc/en-us"
+
+  Scenario: Reporting abuse as a signed-in teacher
+    Given I create a teacher named "Dumbledore"
+    Given I am on "http://studio.code.org/report_abuse"
+    # Email and age are already known and passed to the form
+    And I wait until element "#uitest-email" is not visible
+    And I wait until element "#uitest-age-selector" is not visible
+    And I select the "Other" option in dropdown "uitest-abuse-type"
+    And I type "Mudblood is an offensive term" into "#uitest-abuse-detail"
+    Then I click selector "#uitest-submit-report-abuse" once I see it
+    Then check that the URL matches "https://support.code.org/hc/en-us"
```

Comment:
@epeach When you get a chance, can you take a peek at the UI tests I added for the report abuse form? I test that various account types can successfully submit;  but checking for the alerts when the required fields are left blank was starting to be a time suck. What do you think?